docs/source/zh/perf_train_special.md

```diff
@@ -0,0 +1,58 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# 在Apple芯片(Apple silicon)上进行PyTorch训练
+
+之前，在Mac上训练模型仅限于使用CPU。不过从PyTorch v1.12开始，通过将Apple的Metal性能着色器(MPS)作为后端集成到PyTorch中，[MPS后端](https://pytorch.org/docs/stable/notes/mps.html)能够把PyTorch操作视为自定义的Metal着色器来实现，并部署到`mps`设备上。所以您可以利用Apple芯片的GPU进行模型训练，大幅提升性能和训练速度。
```

Comment:
```suggestion
之前，在 Mac 上训练模型仅限于使用 CPU 训练。不过随着PyTorch v1.12的发布，您可以通过在 Apple Silicon 芯片的 GPU 上训练模型来显著提高性能和训练速度。这是通过将 Apple 的 Metal 性能着色器 (Metal Performance Shaders, MPS) 作为后端集成到PyTorch中实现的。[MPS后端](https://pytorch.org/docs/stable/notes/mps.html) 将 PyTorch 操作视为自定义的 Metal 着色器来实现，并将对应模块部署到`mps`设备上。
```
We have better use "Apple Silicon 芯片"(`Apple Silicon chip`) instead of "Apple 芯片"(`Apple chip`) if Mac with x86_64 Intel CPU doesn't support mps. (Please correct me if I am wrong, because I don't have Mac) :)